### PR TITLE
Add CODEOWNERS for assigning reviewers to PRs

### PR DESCRIPTION
**Pull request by robwalch**
### This PR will...
Add a CODEOWNERS file to the repo

### Why is this Pull Request needed?
Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# See https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file for more details and examples
+
+*       @robwalch
```
Comment (jnatalzia) [inline .github/CODEOWNERS]: never has there been such a true line of code

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: .github/CODEOWNERS (hunk 1 above)
Comment (robwalch) [reply .github/CODEOWNERS]: I didn't want to subject anyone else to the same without discussing first, but feel free to add to it.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# See https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file for more details and examples
+
+*            @robwalch
+/test/unit/  @jnatalzia
+jwplayer/src/css/ @pajong @DanFerrer @robwalch
```
Comment (johnBartos) [inline .github/CODEOWNERS]:
```suggestion
jwplayer/src/js/program @johnBartos 
```

Comment (johnBartos) [reply .github/CODEOWNERS]: Can't get it to insert a newline

**Comment by jnatalzia** [comment]
@robwalch thoughts on merging this now? figure we can always update in future if necessary

**Comment by robwalch** [comment]
@jnatalzia Let's give it a try.

**Pull request merged**